### DISCUSSION TITLE
Project Charts

### DISCUSSION THREAD

**Discussion by Geowil** [community/community#40237] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Is there anyway to make a usable project chart? 

### Body

It seems we are limited to just one set of data.  I am trying to make an actual velocity chart (the one they automatically create is **not** a velocity chart, it doesn't include estimated hours against a custom field for actuals), but I cannot figure out how to get the required data into it.

**Comment by minware-dan** [comment]
Hey @Geowil - 

There isn't a great native solution to this that I am aware of. You could try exporting / pulling the data into a BI tool to build custom reports but you're going to run into some unique challenges depending on how milestones are being used -- DIY velocity calculations will not be as clean or straightforward as they can be with some of the other project management systems out there. 

**Build vs. Buy**: I am a co-founder at minware - we offer an Agile Intelligence tool that recently integrated with Github Issues so my team is very familiar with these challenges. We would be a plug and play option for you to consider if you are open to buying and I would definitely recommend checking out our free trial prior to putting in effort to DIY.  Let me know if we can help.

**Reply by minware-dan** [reply]
Hey @Geowil hope all is well! Curious if you were ultimately able to solve for this?
